contracts/Committee.sol

```diff
@@ -236,7 +242,13 @@ contract Committee is ICommittee, ContractRegistryAccessor, WithClaimableFunctio
 	 * Governance
 	 */
 
-	function setReadyToSyncTimeout(uint48 readyToSyncTimeout) external onlyFunctionalOwner /* todo onlyWhenActive */ {
+	function setMaxTimeBetweenRewardAssignments(uint32 maxTimeBetweenRewardAssignments) external onlyFunctionalOwner /* todo onlyWhenActive */ {
+		require(maxTimeBetweenRewardAssignments > 0, "readyToSyncTimeout must be larger than 0");
```

Comment:
Why limit to != 0. 0 is easy to understand assignment on every transaction. On multiple transactions in the same block the period is 0 so in practice there is no assignment

Reply:
Right, bad c&p :) will remove